tket/tests/TokenSwapping/TableLookup/test_SwapSequenceReductions.cpp

```diff
@@ -0,0 +1,165 @@
+// Copyright 2019-2021 Cambridge Quantum Computing
```

Comment:
For now (this may not be the long-term solution) could you please:
* compile the longer tests conditionally on a `TKET_TESTS_FULL` flag (using `#ifdef` ... `#endif` in the test code);
* modify the `build_and_test.yml` workflow so that when it runs on `schedule` it sets this environment variable when building the tests.
That way they will be run at least once a week.
I can help with the second part if you're not sure how.